src/embind/embind.js

```diff
@@ -1947,6 +1947,8 @@ var LibraryEmbind = {
             // Replace the initial unbound-handler-stub function with the appropriate member function, now that all types
             // are resolved. If multiple overloads are registered for this function, the function goes into an overload table.
             if (undefined === proto[methodName].overloadTable) {
+                // Set argCount in case an overload is registered later
+                memberFunction.argCount = argCount - 2;
```

Comment:
I wonder if this section should also copy `memberFunction.className = classType.name;` over like is done on line 1935 above?